### PR TITLE
adding metric openstack_manila_shares_stuck

### PR DESCRIPTION
**Pull request by talktopiyush**
adding metric openstack_manila_shares_stuck

### CODE REVIEW

**Comment by talktopiyush** [comment]
@rajivmucheli @Carthaca could you please review this pull request.

File: manila/values.yaml [hunk 1]
```diff
@@ -115,7 +115,24 @@ pg_metrics:
         - size_gauge:
             usage: "GAUGE"
             description: "Size of shares"
-
+    openstack_manila_shares_stuck:
+      query: "SELECT share_id, status, host, COUNT(*) FILTER (WHERE updated_at < now() - interval '15 minutes') AS count_gauge, MAX(EXTRACT(epoch FROM now() - updated_at)) AS max_duration_gauge from share_instances where deleted='False' and status in ('deleting','creating') GROUP BY share_id, status, host"
+      metrics:
+        - share_id:
+            usage: "LABEL"
+            description: "Type of the share"
```
Comment (Carthaca) [inline manila/values.yaml]: "Share ID"

File: manila/values.yaml [hunk 2]
```diff
@@ -115,7 +115,24 @@ pg_metrics:
...
+        - status:
+            usage: "LABEL"
+            description: "Share status"
+        - host:
+            usage: "LABEL"
+            description: "Project ID"
```
Comment (Carthaca) [inline manila/values.yaml]: "Share Backend"

**Comment by Carthaca** [comment]
Hi Piyush, besides the nitpicks regarding descriptions: 👍

**Comment by talktopiyush** [comment]
Hey Maurice, updated. 👍

**Pull request merged**